steely/plugins/btc.py

```diff
@@ -1,33 +1,64 @@
 '''Get the latest Bitcoin price (in USD and EUR)'''
 import requests
 
-__author__ = 'CianLR'
+__author__ = ('CianLR', 'byxor')
 COMMAND = 'btc'
 
 API_URL = 'https://api.coindesk.com/v1/bpi/currentprice.json'
 RESP_TEMPLATE = """Bitcoin Price:
-    €{}
-    ${}"""
+€{}
+${}
+Difference since last time: {}"""
 
-def is_valid_response(resp):
-    return ('bpi' in resp and
-            'EUR' in resp['bpi'] and 'rate' in resp['bpi']['EUR'] and
-            'USD' in resp['bpi'] and 'rate' in resp['bpi']['USD'])
+last_euros = 100
```

Comment:
Lemme hear ya say "ALL CAPS"
Reprise: _ALL CAPS_

Reply:
I thought about it, but that implies that `LAST_EUROS` is const. I'll do it anyway.